pages/search_page.py

```diff
@@ -0,0 +1,48 @@
+from .base_page import Page
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.common.by import By
+from .authorized_page import AuthorizedPage
+
+
+class SearchPage(Page):
+    """
+    Page search
+    """
+
+    url = '/search'
+
+    def search_header_form(self):
+        return self.context.driver.find_element_by_css_selector("input.search")
+
+    def input_in_search_top_form(self):
+        self.search_header_form().send_keys('sometext')
+        self.search_header_form().send_keys(Keys.ENTER)
+
+    def search_middle_form(self):
+        return self.context.driver.find_element_by_css_selector("input#q")
+
+    def check_data_from_middle_form(self):
+        if "sometext" == self.search_middle_form().get_attribute("value"):
+            return True
+
+    def user_name(self):
+        return AuthorizedPage.label_page_head(self).text
+
+    def input_correct_data(self):
```

Comment:
Please consider giving correct data as a parameter. Probably this will make the function more flexible and allow it's reusability.

Reply:
At the moment I don't know which data can be better, so I decide do like that. Thank you for review!